src/Filter/modules/TextFilter/textSearchStyles.ts

```diff
@@ -0,0 +1,6 @@
+import styled from '@emotion/styled';
+import Input from '../../../Input/Input';
```

Comment:
Is there a reason for the deeply nested relative import, or can this be an absolute import?  (If you have this as a matter of philosophy, no worries; internally we usually discourage it.)

Reply:
It can be an absolute import, this was probably just how it was auto-imported. I don't think we've really made a pass at solidifying our style one way or the other, but I would prefer absolute imports.

Reply:
The issue was actually a jest one. When I had it as an absolute import, the tests began failing. I'm sure it's a simple config change, but I didn't look to deeply into it since it wasn't the focus of this PR. I'd prefer to keep as is for now and raise a separate issue to address it throughout the library. Thoughts @adorack-tri, @JonTheBurley?

Reply:
Huh, interesting.  I'm fine with your proposal to not chase down Jest configuration issues right this moment.

Reply:
Issue #96 was created. Thanks @JonTheBurley for the likely solution!